### PR TITLE
10.0 mig sale order line variant description

### PR DESCRIPTION
**Pull request by angedras**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together all consecutive "OCA Transbot" commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by pedrobaeza** [comment]
I think that maybe we should move this to OCA/product-variant. What do you think, @tafaRU ?

**Comment by tafaRU** [comment]
@angedras, please address https://github.com/OCA/sale-workflow/pull/462#issuecomment-299238722

File: sale_order_line_variant_description/README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+   :alt: License: AGPL-3
+
+===================================
+Sale order line variant description
+===================================
+
+This module allows to specify different sale description values for each
+product variant and use them on the sale order lines as their names.
+
+Installation
```
Comment (tafaRU) [inline sale_order_line_variant_description/README.rst]: Installation, Configuration, Known issues / Roadmap can be removed as useless.

File: sale_order_line_variant_description/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# © 2015 Alex Comba - Agile Business Group
```
Comment (tafaRU) [inline sale_order_line_variant_description/__init__.py]: Please replace by
```
# Copyright 2015-2017 Alex Comba - Agile Business Group
```
according to https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2

File: sale_order_line_variant_description/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2015 Alex Comba - Agile Business Group
```
Comment (tafaRU) [inline sale_order_line_variant_description/models/__init__.py]: Same as above.

File: sale_order_line_variant_description/views/product_view.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Alex Comba - Agile Business Group
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+     
+<openerp>
+    <data>
```
Comment (tafaRU) [inline sale_order_line_variant_description/views/product_view.xml]: Please remove data tag.
See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0 as reference.

**Review by tafaRU — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
@angedras please, if you attend comment we are in OCA code sprint, we can review you! Please!

**Comment by tafaRU** [comment]
I close this as superseded by https://github.com/OCA/product-variant/pull/62

**Pull request closed without merging**